README.md

```diff
@@ -34,9 +34,20 @@ You can configure the default behavior through various settings in your `setting
 }
 ```
 
+You can also set a new keyboard shortcut for the command in `keybindings.json`:
+
+```json
+[
+  {
+    "key": "cmd+n",
+    "command": "extension.createNewFile"
```

Comment:
This is just a mistake on my side. For all the settings I moved away from the extension namespace. I'll modify this in the next update.

Thanks!